package.json

```diff
@@ -268,6 +269,56 @@
         "./vsdbg"
       ],
       "installTestPath": "./.debugger/vsdbg-ui"
+    },
+    {
+      "description": "Razor Language Server (Windows / x64)",
+      "url": "https://razorvscodetest.blob.core.windows.net/languageserver/RazorLanguageServer-win-x64-1.0.0-alpha1-20180924.3.zip",
```

Comment:
These URLs will need to change before this ships. We want the downloads to come from a particular CDN which we're still getting access to.